sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountIf.scala

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Can we do `RuntimeReplaceable` like `Every` or `Any`?

Reply:
Sorry, I don't understand your intention. Is there necessary to use `RuntimeReplaceable` in this case?

Reply:
Not necessary but in that way we can just rewrite and reuse the expressions (take a look for them). `Every`, `Any`, `Some` are similar instances like this case but they were rewritten as `Max` and `Min` . Let's follow that way.